### ISSUE TITLE
App Crashed

### ISSUE THREAD

**Issue by nonamejan** [benmotyka/crypto-checker#2]
npm run dev worked but now I have this error

![Screenshot_13](https://user-images.githubusercontent.com/113781755/211489613-bce59265-c00d-4743-962d-3e98767bfff5.png)

**Comment by benmotyka** [comment]
@nonamejan have you installed npm packages? `npm install`

**Comment by nonamejan** [comment]
yes I did. I also reinstall everything and it still doesn't work

**Comment by benmotyka** [comment]
@nonamejan can you provide me with a output of `npm install` command?

**Comment by nonamejan** [comment]
Ok maybe my os (windows) is the issue

![Screenshot_14](https://user-images.githubusercontent.com/113781755/211505290-8ba95876-c469-448d-a69b-84eb205f83ec.png)

**Comment by benmotyka** [comment]
@nonamejan I will try to test it on windows machine soon

**Comment by nonamejan** [comment]
Ok great :)
